content/bn/software_as_a_service.md

```diff
@@ -0,0 +1,19 @@
+---
+Title: সফ্টওয়্যার এজ এ সার্ভিস(Software as a Service)(SaaS)
+Status: Completed
+Category: প্রযুক্তি
+---
+
+## এটা কি
+
+সফ্টওয়্যার এজ এ সার্ভিস (SaaS) ব্যবহারকারীদের ইন্টারনেটের মাধ্যমে ক্লাউড-ভিত্তিক পরিষেবাগুলির সাথে সংযোগ হতে এবং ব্যবহার করতে দেয়৷ সাধারণ উদাহরণ হল ইমেইল, ক্যালেন্ডারিং এবং অফিস টুল (যেমন Gmail, Amazon Web Services, GitHub, Slack)। সফ্টওয়্যার এজ এ সার্ভিস (SaaS) সম্পূর্ণ সফ্টওয়্যার সেবা প্রদান করে যা ব্যবহারকারী পে-এজ-ইউ-গো (Pay-as-you-go) অর্থাৎ যতটুকু সেবা গ্রহণ করা হবে ঠিক ততটুকুর অর্থ প্রদান এই ভিত্তিতে ব্যবহার করেন। সমস্ত অপারেশন এবং রক্ষণাবেক্ষণের কাজ এবং অ্যাপ্লিকেশন ডেটা পরিষেবা প্রদানকারী দ্বারা পরিচালিত হয়।
+
+## এটা যেসব সমস্যাতে দৃষ্টিপাত করে
+
+প্রথাগতভাবে, ব্যবসায়িক সফ্টওয়্যারগুলো পৃথক কম্পিউটারে ইনস্টল করা হয়, যার রক্ষণাবেক্ষণ এবং আপডেট করার জন্য একজন প্রশাসকের প্রয়োজন হয়। উদাহরণ স্বরূপ: একটি প্রতিষ্ঠান গ্রাহক চাহিদা ব্যবস্থাপনা (CRM) এর জন্য স্ব-শরীর(on-premise) সফ্টওয়্যার ব্যবহার করতে পারে। এই সফ্টওয়্যারটি অভ্যন্তরীণ আইটি বিভাগ নিয়োগ করে ক্রয়, ইনস্টল, সুরক্ষা, রক্ষণাবেক্ষণ এবং নিয়মিত আপগ্রেড করা প্রয়োজন, যা আইটি টিমের উপর একটি বোঝাস্বরূপ ৷ লাইসেন্স, ইন্সটলেশন এবং সম্ভাব্য অতিরিক্ত হার্ডওয়্যারের সাথে যুক্ত আপ ফ্রন্ট খরচ নিষিদ্ধ হতে পারে। চাহিদার প্রতি সাড়া দেওয়াও কঠিন হতে পারে এবং [scale](/scalability/) বৃদ্ধি বা পরিবর্তনের প্রতিক্রিয়ায় দ্রুত প্রয়োজন অনুযায়ী উপরে ও নিচে যাতায়াত সম্ভব না হতে পারে।
```

Comment:
Would it be better to use the Bengali word of "scale", rather than using the English word "scale" as-is?

For example, in [`cloud_computing.md` of this PR](https://github.com/cncf/glossary/pull/848/files#diff-4ce133fe6b2215efd14636f576fee90fcbea9ba9606ecb7a45fd59bc6b5587af),
I can see:
`[স্কেল(scale)](/scalability/)`